test/bridge/slippages-test.ts

```diff
@@ -0,0 +1,65 @@
+// BEFORE: :file:`src/bridge/slippages.ts`
```

Comment:
@BlazeWasHere in general, try to strip stuff like this out before a merge

Reply:
Rather, while I'd prefer it not be in _code_ files, I do not mind -- actually, I highly encourage it -- if you put coverage details in a different file at the same level as the coverage-checked file.